funnel/views/project.py

```diff
@@ -281,6 +299,42 @@ def rsvp(self):
     def rsvp_list(self):
         return {'project': self.obj, 'statuses': RSVP_STATUS}
 
+    @route('save', methods=['POST'])
+    @render_with(json=True)
+    @lastuser.requires_login
+    @requires_permission('view')
+    def save(self):
+        form = ProjectSaveForm()
+        if form.validate_on_submit():
+            proj_save = SavedProject.query.filter_by(user=current_auth.user, project=self.obj).first()
+            if form.save.data:
+                if proj_save is None:
+                    proj_save = SavedProject(user=current_auth.user, project=self.obj)
+                    form.populate_obj(proj_save)
+                    db.session.commit()
+                    return {'status': 'success'}
```

Comment:
The valid values for status (by convention) are `ok` and `error`. [We've been using it a while](https://github.com/hasgeek/lastuser/blame/18e3af9259dd81913d9c70a725eaed19fba09d05/lastuser_oauth/views/resource.py#L114).